homeassistant/components/netatmo/sensor.py

```diff
@@ -221,37 +246,31 @@ class NetatmoSensorEntityDescription(SensorEntityDescription):
     ),
     NetatmoSensorEntityDescription(
         key="reachable",
-        name="Reachability",
         netatmo_name="reachable",
         entity_registry_enabled_default=False,
         entity_category=EntityCategory.DIAGNOSTIC,
-        icon="mdi:signal",
     ),
```

Comment:
This one just returns `True` and `False` or `None`, and should just be a binary_sensor tbh